ui/src/alerting/components/CheckCard.tsx

```diff
@@ -47,10 +47,8 @@ const CheckCard: FunctionComponent<Props> = ({
   const onClone = () => {}
 
   const onToggle = () => {
-    const status =
-      check.status == CheckBase.StatusEnum.Active
-        ? CheckBase.StatusEnum.Inactive
-        : CheckBase.StatusEnum.Active
+    const status = check.status === 'active' ? 'inactive' : 'active'
```

Comment:
I really like that we can compare enum values to their string values.

Reply:
Me too! It's a small thing but somehow so nice